xml/ay_bigfile.xml

```diff
@@ -17169,7 +17169,7 @@ echo -n 100
          <literal>autoyast=tftp://..</literal>, or
          <literal>autoyast=cifs://..</literal>.
          In order to have network access the network will has to be defined via the
-         boot options in linuxrc. One simplest way would to do define it via DHCP like:
+         boot options in linuxrc. The simplest way would to do define it via DHCP like:
```

Comment:
would **be to** define - you missed that part

Reply:
How about this instead: "Network access needs to be defined using the boot options in linuxrc. This can be done via DHCP:"

* shorter
* removes the ungrammatical use of "will"
* makes no promises about simplicity (how simple something is is for the user to decide anyway)

Reply:
Thanks, Stefan - I agree and changed it accordingly.

Reply:
"The network will has to" ..."The simplest way would to do"? 
Surely:
"The network has to"... "would be to..."